<launch>
  <arg name="base_global_planner" default="navfn/NavfnROS"/>
  <arg name="base_local_planner" default="dwa_local_planner/DWAPlannerROS"/>

  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <remap from="cmd_vel" to="/markhor/tracks/diff_drive_controller/cmd_vel"/>
    <remap from="odom" to="/markhor/tracks/diff_drive_controller/odom"/>

    <param name="base_global_planner" value="$(arg base_global_planner)"/>
    <param name="base_local_planner" value="$(arg base_local_planner)"/>  
    <rosparam file="$(find markhor_2dnav)/config/planner.yaml" command="load"/>

    <rosparam file="$(find markhor_2dnav)/config/costmap_common.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find markhor_2dnav)/config/costmap_common.yaml" command="load" ns="local_costmap" />

    <rosparam file="$(find markhor_2dnav)/config/costmap_local.yaml" command="load" ns="local_costmap" />
    <param name="local_costmap/width" value="10.0"/>
    <param name="local_costmap/height" value="10.0"/>


    <rosparam file="$(find markhor_2dnav)/config/costmap_global_laser.yaml" command="load" ns="global_costmap"/>
    <param name="global_costmap/width" value="100.0"/>
    <param name="global_costmap/height" value="100.0"/>
  </node>

</launch>